modules/geo-layers/src/index.ts

```diff
@@ -35,7 +35,7 @@ export {default as GeohashLayer} from './geohash-layer/geohash-layer';
 export {default as _GeoCellLayer} from './geo-cell-layer/GeoCellLayer';
 
 // Types
-export type {ImageryLayerProps as _ImageryLayerProps} from './imagery-layer/imagery-layer';
+export type {WMSLayerProps as _WMSLayerProps} from './wms-layer/wms-layer';
```

Comment:
I don't think we do underscore for types

Reply:
Will remove but look at line 48 in same file